app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -3096,8 +3098,10 @@ class BrowserTabFragment :
     private fun showKeyboard() {
         if (!isHidden) {
             Timber.v("Keyboard now showing")
-            binding.legacyOmnibar.omnibarTextInput.postDelayed(KEYBOARD_DELAY) { binding.legacyOmnibar.omnibarTextInput?.showKeyboard() }
-            binding.legacyOmnibar.omniBarContainer.isPressed = true
+            with(binding.legacyOmnibar.omnibarTextInput) {
+                showKeyboard(this)
+                isPressed = true
```

Comment:
it’s not the omnibarTextInput that has the isPressed state as true. 

```suggestion
                binding.legacyOmnibar.omniBarContainer.isPressed = true
```

Reply:
Fixed! Thanks 🙌